shopinvader_product_seasonality/components/seasonal_config_line_event_listener.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Camptocamp SA
+# @author: Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.component.core import Component
+from odoo.addons.component_event import skip_if
+
+
+class SeasonalConfigLineEventListener(Component):
+    _name = "seasonal.config.line.event.listener"
+    _inherit = "base.event.listener"
+
+    _apply_on = ["seasonal.config.line"]
+
+    @skip_if(lambda self, record, **kwargs: not record.product_id.shopinvader_bind_ids)
```

Comment:
Maybe I'm wrong, but seems you invert the condition.
You want to skip if the binding exists:
```suggestion
    @skip_if(lambda self, record, **kwargs: record.product_id.shopinvader_bind_ids)
```